components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/entitlement/XacmlEntitlementServiceImpl.java

```diff
@@ -187,12 +187,12 @@ public void generateAndSaveEntitlementPolicies(List<XACMLPolicyTemplateContext>
     }
 
     @Override
-    public boolean isPermitted(EntitlementDecisionRequest request) {
+    public boolean isPermitted(EntitlementDecisionRequest request) throws AppManagementException {
```

Comment:
Formatting issues and Java doc comment issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files